src/utils/auth-utils.js

```diff
@@ -1,17 +1,23 @@
-const db = require('../models');
-const User = db.User;
+const db = require("../models");
+const { User, Address, Booking, Charger, Vehicle } = db;
 const Op = db.Sequelize.Op;
 
-async function findUser(email) {
+async function findUser(username) {
   try {
-    const user = await User.findOne({ where: { email: `${email}` } });
-    return user;
+    const user = await User.findOne({
+      where: { username },
+      include: Address,
+      // [{ model: Address }, { model: Booking }, { model: Charger }, {model: Vehicle}],
+    });
+    return user.dataValues;
   } catch (err) {
-    res.status(500);
-    return res.json({ error: err.message });
+    console.log(err.message)
+
+    // res.status(500);
+    // return res.json({ error: err.message });
```

Comment:
We're logging an error message here, but not returning an error if a User isn't found. Can this be changed?

Reply:
Oh i was gonna handle "user/charger not found" error in the controller instead of utils

Reply:
All good if that's being changed 😄